defactor-com/ui-kit
# Feat/enhance chart component 72

## Enhance Dashboard Component

### What does this PR do?

- Resolve #72 
- Resolve #76 
- Allow to have as many lines and bottom items as needed
- Allow to enable and disable the lines in the chart
- Allow to pass a function to format the numbers
- Add a CardComponent to replace FlatComponent in the Dashboard
- Remove currency label from the cards

### Steps to test

1. Run the project locally
2. Go to Dashboard component section
3. Check the Component

### Screenshots

**Desktop**

![imagen](https://github.com/defactor-com/ui-kit/assets/66583677/64a790f2-3c36-4593-9a52-70bb19f35e8e)
![imagen](https://github.com/defactor-com/ui-kit/assets/66583677/19cfe06a-5321-4a26-ae06-e32433928b53)

**Tablet**

![imagen](https://github.com/defactor-com/ui-kit/assets/66583677/2fa80979-a3fc-4093-a9db-73171b65b0b1)

**Mobile**

![imagen](https://github.com/defactor-com/ui-kit/assets/66583677/8cda3974-78e3-40d5-98e9-d1b66f053a04)

### Label Issue

**Before**
![imagen](https://github.com/defactor-com/ui-kit/assets/66583677/0361a2e3-adf4-403f-9e5d-49b792aad7e7)
**After**
![imagen](https://github.com/defactor-com/ui-kit/assets/66583677/e2ebe40d-14bc-4e95-9a8a-ed59be2be814)
